content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -144,9 +144,432 @@ dependencies {
+| Zipkin Exporter       | otel.exporter.zipkin.enabled                | `true`        | ZipkinSpanExporter        |
+| Logging Exporter      | otel.exporter.logging.enabled               | `false`       | LoggingSpanExporter       |
+
+<!-- Slf4j Log Correlation  otel.springboot.loggers.slf4j.enabled		true   		org.slf4j.MDC -->
+
+
+### Enabling/Disabling Features
+
+| Feature               | Property                                    | Default Value | ConditionalOnClass        |
+|-----------------------|---------------------------------------------|---------------|---------------------------|
+| spring-web            | otel.instrumentation.spring-webmvc.enabled  | `true`        | RestTemplate              |
+| spring-webmvc         | otel.instrumentation.spring-web.enabled     | `true`        | OncePerRequestFilter      |
+| spring-webflux        | otel.instrumentation.spring-webflux.enabled | `true`        | WebClient                 |
+| @WithSpan             | otel.instrumentation.annotations.enabled    | `true`        | WithSpan, Aspect          |
+
+### Resource Properties
+
+| Feature  | Property                                                            | Default Value          |
+| -------- |---------------------------------------------------------------------| ---------------------- |
+| Resource | otel.springboot.resource.enabled                                    | `true`                 |
+|          | otel.resource.attributes (old: otel.springboot.resource.attributes) | `empty map`            |
+
+`otel.resource.attributes` supports a pattern-based resource configuration in the
+application.properties like this:
+
+```
+otel.resource.attributes.environment=dev
+otel.resource.attributes.xyz=foo
+```
+
+It's also possible to specify the resource attributes in `application.yaml`:
+
+```yaml
+otel:
+  resource:
+    attributes:
+      environment: dev
+      xyz: foo
+```
+
+Finally, the resource attributes can be specified as a comma-separated list, as described in the
+[specification](https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_resource_attributes):
+
+```shell
+export OTEL_RESOURCE_ATTRIBUTES="key1=value1,key2=value2"
+```
+
+The service name is determined by the following precedence, in accordance with the OpenTelemetry
+[specification](https://opentelemetry.io/docs/concepts/sdk-configuration/general-sdk-configuration/#otel_service_name):
+
+1. `otel.service.name` spring property or `OTEL_SERVICE_NAME` environment variable (highest
+   precedence)
+2. `service.name` in `otel.resource.attributes` system/spring property or `OTEL_RESOURCE_ATTRIBUTES`
+   environment variable
+3. `service.name` in `otel.springboot.resource.attributes` system/spring property
+4. `spring.application.name` spring property
+5. the default value `unknown_service:java` (lowest precedence)
+
+##### Exporter Properties
```

Comment:
I would include the `Exporter Properties` part in the `## OpenTelemetry Spring starter`, apart from the _Jaeger Exporter_ and _Zipkin Exporter_ features the I would include in the _### Spring autoconfiguration_ part.

Reply:
I moved these properties, please let me know If they ended up in the right places